### ISSUE TITLE
cocoapod update SDWebImage 4.0，I don't find 'FLAnimatedImageView+WebCache' file?

### ISSUE THREAD

**Issue by asd8855** [rs/SDWebImage#1796]
### New Issue Checklist

* [x] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [x] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [x] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. ios / tvos
 Platform Version        | e.g. 8.0
 SDWebImage Version      | e.g. 3.7.6
 Integration Method      | e.g. carthage / cocoapods / manually
 Xcode Version           | e.g. Xcode 7.3
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

**Comment by asd8855** [comment]
platform: ios,'8.0'

**Comment by asd8855** [comment]
XCode:8.2.1

**Comment by asd8855** [comment]
sorry,this is my question

**Comment by skyline75489** [comment]
Glad to hear it. I'll close this issue then.
